Source/DataSources/KmlDataSource.js

```diff
@@ -1150,7 +1150,7 @@ define([
                 entity.corridor = corridor;
                 corridor.positions = coordinates;
                 if (defined(polyline)) {
-                    corridor.material = defined(polyline.material) ? polyline.material.color : Color.WHITE;
+                    corridor.material = defined(polyline.material) ? polyline.material.color.getValue() : Color.WHITE;
```

Comment:
`getValue` requires `time` as its first parameter, if we are 100% sure that `.material.color` will be a constant value (maybe it's impossible to make it time varying with kml?) then we should still use `Iso860.MINIMUM_VALUE` here.  @tfili can you check if this is indeed always constant.